src/HtmlUtils.tsx

```diff
@@ -635,6 +638,57 @@ export function linkifyAndSanitizeHtml(dirtyHtml: string, options = linkifyMatri
     return sanitizeHtml(linkifyString(dirtyHtml, options), sanitizeHtmlParams);
 }
 
+const getAbsoluteUrl = (() => {
+    let a: HTMLAnchorElement;
+
+    return (url: string) => {
+        if (!a) {
+            a = document.createElement('a');
+        }
+        a.href = url;
+        return a.href;
+    };
+})();
```

Comment:
this could do with a bit of documentation to say what it's doing, and why it gets its own scope.